src/main/java/Deadline.java

```diff
@@ -0,0 +1,27 @@
+public class Deadline extends Task{
+    private static final String DEADLINE_TYPE = "D";
```

Comment:
I like how you followed the coding standards and define "D" as a final string constant. I see this in other places as well.